docs/organizations/managing-security-and-risk.md

```diff
@@ -1,73 +1,131 @@
+![Security and risk management open finding distribution panel](images/security-risk-management-overview-distribution.png)<!-- TODO TAROT-2591: Screenshot -->
 
-![Security and risk management items](images/security-risk-management-item-list.png)
+### Open finding history
 
-## Exporting the security item list
+The **Open finding history** panel displays a graph showing weekly trends of open findings from the past three months, grouped by severity.
+
+For a detailed view of the distribution on a specific week, hover over the graph.
+
+![Security and risk management open finding history panel](images/security-risk-management-overview-history-open.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Activity history
+
+The **Activity history** panel displays a graph showing weekly counts of open and closed findings over the past three months and the overall open finding trend.
+
+For a detailed view of the counts on a specific week, hover over the graph.
+
+![Security and risk management activity history panel](images/security-risk-management-overview-history-activity.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Top 10 high-risk repositories
+
+The **Top 10 high-risk repositories** panel displays the repositories with the highest number of open findings, ordered by severity.
+
+!!! note
+    This panel may list less than ten repositories if there are fewer than ten repositories with open findings in the organization or if less than ten repositories are selected in the dropdown filter.
+
+![Security and risk management high risk repositories panel](images/security-risk-management-overview-top-risk.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Top 10 common security categories
+
+The **Top 10 common security categories** panel displays the most common security categories of open findings, ordered by count.
+
+To access the findings page with the corresponding filter applied, click on a category.
+
+![Security and risk management top categories panel](images/security-risk-management-overview-top-categories.png)<!-- TODO TAROT-2591: Screenshot -->
+
+## Findings {: id="item-list"}
+
+The **Security and risk management findings** page displays a filtered list of findings, sorted by status. Use this page to review and prioritize findings and track the progress of your security efforts.
+
+To access the overview page, select an organization from the top navigation bar, select **Security and risk** on the left navigation sidebar, and click the **Findings** tab in the top bar.
+
+![Security and risk management findings page](images/security-risk-management-findings.png)<!-- TODO TAROT-2591: Screenshot -->
+
+When viewing the findings, you can update the filtering criteria by clicking the **Severity**, **Status**, **Repository**, **Security category**, or **Scan type** drop-downs above the list.
+
+### Viewing a single finding
```

Comment:
Do we need a section here? 🤔

Reply:
We'll have a new single-finding detail view with some information that may be of use. It's more of a placeholder for now.

Reply:
I'll remove this for now; it will be implemented later.